service/src/main/java/org/mskcc/smile/service/impl/TempoMessageHandlingServiceImpl.java

```diff
@@ -165,4 +236,29 @@ public void onMessage(Message msg, Object message) {
             }
         });
     }
+
+    private void setupQcCompleteHandler(Gateway gateway,
+            TempoMessageHandlingService tempoMessageHandlingService) throws Exception {
+        gateway.subscribe(TEMPO_WES_QC_COMPLETE_TOPIC, Object.class, new MessageConsumer() {
+            @Override
+            public void onMessage(Message msg, Object message) {
+                try {
+                    String qcCompleteJson = mapper.readValue(
+                        new String(msg.getData(), StandardCharsets.UTF_8),
+                        String.class);
+                        Map<String, String> qcCompleteMap = mapper.readValue(qcCompleteJson, Map.class);
+                        QcComplete qcComplete = new QcComplete(qcCompleteMap.get("date"),
```

Comment:
Not directly related to `QcComplete` but about `MafComplete`. I noticed that one of its JSON property names is `normal_primaryId`. Do you know if there is a functional reason behind why we're mixing snake and camel cases together, or was it simply a stylistic choice?

Reply:
Asking Anne Marie if we can change it

Reply:
Thanks!